test/tests/functional/pbs_pbsnodes_output_trimmed.py

```diff
@@ -83,8 +83,8 @@ def test_pbsnodes_output(self):
         res = rc['out']
         self.assertEqual(res[0].split(" ")[0], hname)
         self.server.manager(MGR_CMD_DELETE, NODE, None, '')
-        
+
     def tearDown(self):
-        if self.is_host_file_changed == True:
-            with open("/etc/hosts", "w")as fd:
+        if self.is_host_file_changed:
```

Comment:
Minor comment: I'd say if self.is_host_file_changed is True: but this works as well.  Up to you if you want to change it.